tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,33,34,35,36,37,38,39}, py{35,36,37,38,39}-travis, docs, flake
+envlist = py{27,33,34,35,36,37,38,39,310,311}, py{35,36,37,38,39,310,311}-travis, docs, flake
```

dropped python2.7 support

```suggestion
envlist = py{37,38,39,310,311}, py{37,38,39,310,311}-travis, docs, flake
```